packages/beacon-node/src/monitoring/system.ts

```diff
@@ -93,6 +99,8 @@ class System {
   }
 
   private async collectDiskData(): Promise<void> {
+    // Usage of `system.fsSize()` is more readable than `fsSize()`
+    // eslint-disable-next-line import/no-named-as-default-member
```

Comment:
Can you disable this rule for the whole file and remove all these individual comments?